app/views/layouts/application.html.haml

```diff
@@ -32,6 +32,13 @@
           .cads-grid-container
             .cads-grid-row
               %main.cads-grid-col-md-12
+                %svg{ width: "17", height: "14", viewBox: "0 0 17 15", style: "display: none;", xmlns: "http://www.w3.org/2000/svg" }
+                  %g{ id: "halfStar" }
```

Comment:
👃🏻 being a nosey p - what's the reason for keeping the SVG element over here instead of the component haml?  Or is this the fix for @marianayovcheva's comment?

Reply:
Not nosey at all! 😄 

The `id: "paint0_linear_658_12679"` appeared everywhere that the half star component rendered, which was a WCAG fail as it was a duplicate id. This was the lowest place I could see that I could put the svg so that it would be available everywhere it was needed, but so that the id wouldn't be duplicated. Very happy to change it if there's a better way of doing it!